invoices/invoiceregistry_test.go

```diff
@@ -0,0 +1,176 @@
+package invoices
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+var (
+	testTimeout = 5 * time.Second
+
+	preimage = lntypes.Preimage([32]byte{
```

Comment:
fwiw, we should be able to instantiate this directly using
```golang
preimage = lntypes.Preimage{
    ...
}
```
not a blocker tho

Reply:
fixed as i needed to rebase anyway